Update i18n GUIDELINES (#103959)
elastic/kibana: packages/kbn-i18n/GUIDELINE.md

- AriaLabel (if it's `aria-label` tag attribute),
- ErrorMessage (if it's an error message),
- LinkText (if it's `<a>` tag),
- ToggleSwitch
- `.markdown` (if it's markdown)

> For markdown, do not include the actual markdown identifiers `#`, `*`, `_` etc.

Author: Remove this

  If this group of sentences is separated it’s possible that the context of the `'it'` in `'close it'` will be lost.

### Large paragraphs

> Try to avoid having to use large paragraphs of text. These are difficult to maintain and often need slight changes when the information they portray becomes out of date. 

Reviewer: ```suggestion
Try to avoid using large paragraphs of text. These are difficult to maintain and often need slight changes when the information they portray becomes out of date. 
```

> If you have absolutely no choice, you can split paragraphs into a _few_ i18n chunks whenever it makes sense to do so. Similar principles of splitting sentences apply here. Each chunk on its own should convey the context of the information.

Author: ```suggestion
If you have absolutely no choice, you can split paragraphs into a _few_ i18n chunks. Chunks should be split at logical points to ensure they contain enough context to be intelligible on their own.
```

> If you have absolutely no choice, we can split paragraphs into a _few_ i18n chunks whenever it makes sense to do so. Similar principles of splitting sentences apply here. Each chunk on it's own should convey the context of the information.

Reviewer: ```suggestion
If you have absolutely no choice, you can split paragraphs into a _few_ i18n chunks whenever it makes sense to do so. Similar principles of splitting sentences apply here. Each chunk on its own should convey the context of the information.
```

> Each chunk on it's own should convey the context of the information

nit: Had to read this a couple times to understand the intent. Maybe something along these lines?  "Chunks should be split at logical points to ensure they contain enough context to be intelligible on their own."

Author: @lukeelmers Thanks for the suggestion!
